buddy_init/base_skeleton/src/config.py

```diff
@@ -0,0 +1,28 @@
+import math
```

Comment:
We should use the mp friendly config for the tutorial
i.e. 
main.py:
```python
import config
import buddy


buddy.register(vars(config))
tb = buddy.deploy()
main(tb)
```

Reply:
imho rm.

Reply:
rm